docker/standalone/docker-compose.yml

```diff
@@ -8,15 +8,16 @@ networks:
       name: ortelius_services
 services:
   avalanche:
+    env_file:
```

Comment:
If we name the file `.env` it will be more conventional and we can remove this line.

https://docs.docker.com/compose/environment-variables/#the-env-file

Reply:
yea I thought about that..  i kinda liked a specific named file.  more specific..  But I can go with .env instead.